Telemetry/Telemetry.swift

```diff
@@ -18,6 +18,7 @@ public class Telemetry {
     private let scheduler: TelemetryScheduler
     private var beforeSerializePingHandlers = [String: [BeforeSerializePingHandler]]()
     private var pingBuilders: [String : TelemetryPingBuilder]
+    private var backgroundTasks: [String : UIBackgroundTaskIdentifier]
```

Comment:
nit: this and above it could be `private var thing = [String: type]()`

Reply:
I always forget that Swift lets you do that. Thanks.